src/cloud/project/project-patch.md

```diff
@@ -35,37 +29,165 @@ All patch file names must end with the `.patch` extension.
+   -  Deprecated—The individual patch is marked as deprecated by Magento and we recommend reverting it if you have applied it. After you revert a deprecated patch, it will no longer be displayed in the status table.
+   -  Custom—All patches from the 'm2-hotfixes' directory.
+
+-  **Status**:
+
+   -  Applied—The patch has been applied.
+   -  Not applied—The patch has not been applied.
+   -  N/A—The status of the patch cannot be defined due to conflicts.
+
+-  **Details**:
+
+   -  Affected components—The list of affected Magento modules.
+   -  Required patches—The list of required patches (dependencies).
+   -  Recommended replacement—The patch that is a recommended replacement for a deprecated patch.
+
+## Apply a patch in a local environment
 
 You can apply patches manually in a local environment and test them before you deploy.
 
 {:.procedure}
-To apply {{site.data.var.ece}} patches manually:
+To apply patches in a local environment:
+
+1. Add the 'QUALITY_PATCHES' variable to the `.magento.env.yaml` file and list the required patches underneath.
+
+   ```yaml
+   stage:
+       build:
+           QUALITY_PATCHES:
+               - MCTEST-1002
+               - MCTEST-1003
+   ```
 
 1. From the project root, apply the patches.
 
    ```bash
    php ./vendor/bin/ece-tools patch
    ```
 
+   Patches will be applied in the following order:
+
+   -  Required patches
+   -  Optional individual patches
+   -  Custom patches from the `/m2-hotfixes` directory
+
 1. Clear the Magento cache.
 
    ```bash
    php ./bin/magento cache:clean
    ```
 
-   You can also clean the cache using the [Magento Admin Cache Management](http://docs.magento.com/m2/ee/user_guide/system/cache-management.html).
-
 1. Test the patches, make any necessary changes to custom patches.
 
+## Apply a patch in a Cloud environment
+
+{:.bs-callout-warning}
+We strongly recommend testing all patches in a staging or development environment before deploying to production.
+
+{:.procedure}
+To apply patches in a Cloud environment:
+
+1. Add the 'QUALITY_PATCHES' variable to the `.magento.env.yaml` file and list the required patches underneath.
+
+   ```yaml
+   stage:
+       build:
+           QUALITY_PATCHES:
+               - MCTEST-1002
+               - MCTEST-1003
+   ```
+
+1. Commit and push the updated `.magento.env.yaml` file:
+
+   ```bash
+   git commit -m "Apply patch"
+   ```
+
+   ```bash
+   git push origin <branch-name>
+   ```
+
 ## Apply a custom patch
 
-When you deploy, `{{site.data.var.ct}}` applies all Magento patches and any custom patches that you add to the `/m2-hotfixes` directory in the Magento project root.
+When you deploy, `{{ site.data.var.ct }}` applies all Magento patches and any custom patches that you add to the `/m2-hotfixes` directory in the Magento project root.
 
 {:.procedure}
-To apply and test a custom patch:
+To apply and test a custom patch on a cloud environment:
```

Comment:
In other places you used different capitalization.

Reply:
Fixed in 8b0c2def7b7de9734dda528d0139bccf6a3fabad